### ISSUE TITLE
support multiple gorm

### ISSUE THREAD

**Issue by philhuan** [acmestack/gorm-plus#79]
We need to connect to multiple databases in our project, but gplus's gorm as a global variable.
can gplus support multi gorm??

**Comment by afumu** [comment]
Good idea, we can add it later, or if you're interested, you can submit pr

**Comment by philhuan** [comment]
In actual projects, multiple data sources must be used concurrently at the same time, so the value of global variables cannot be switched through one method. 

We can provide API of gplus through the methods of an object, and then distinguish different data sources through the attribute values ​​​​of the object.

For example：
```

gorm1Db, _ = gorm.Open()
gorm2Db, _ = gorm.Open()

gplus1Db = gplus.New(gorm1Db)
gplus2Db = gplus.New(gorm2Db)


users1, resultDb := gplus1Db.SelectList[User](nil)
users2, resultDb := gplus2Db.SelectList[User](nil)

```

However, this will change the API of gplus.

**Comment by timandy** [comment]
> 
> For example：
> 
> ```
> 
> gorm1Db, _ = gorm.Open()
> gorm2Db, _ = gorm.Open()
> 
> gplus1Db = gplus.New(gorm1Db)
> gplus2Db = gplus.New(gorm2Db)
> 
> 
> users1, resultDb := gplus1Db.SelectList[User](nil)
> users2, resultDb := gplus2Db.SelectList[User](nil)
> ```


 
So far, go doesn't support generic methods.
